roles/nuage_ca/tasks/main.yaml

```diff
@@ -40,7 +40,7 @@
   delegate_to: "{{ nuage_ca_master }}"
 
 - name: Create CA crt
-  command: openssl req -new -x509 -key "{{ nuage_ca_key }}" -out "{{ nuage_ca_crt }}" -subj "/CN=nuage-signer"
+  command: openssl req -new -x509 -key "{{ nuage_ca_key }}" -out "{{ nuage_ca_crt }}" -subj "/CN=nuage-signer" -days {{ nuage_mon_cert_validity_period }}
```

Comment:
I see default for `nuage_mon_cert_validity_period` is set in `nuage_common`, but not in `nuage_ca` (which is being run later on).

I guess `nuage_mon_cert_validity_period` should be moved to `nuage_ca` defaults

Reply:
Thank you for the review. `nuage_common` is supposed to be run before `nuage_ca`. I made changes to address this.